src/SeleniumLibrary/__init__.py

```diff
@@ -32,6 +32,7 @@
                                       TableElementKeywords,
                                       WaitingKeywords)
 from SeleniumLibrary.locators import ElementFinder
+from SeleniumLibrary.locators import TableElementFinder
```

Comment:
Could (and IMHO should) be combined with the above import:

    from SeleniumLibrary.locators import ElementFinder, TableElementFinder